torch/csrc/distributed/autograd/context/dist_autograd_container.cpp

```diff
@@ -5,56 +5,94 @@ namespace torch {
 namespace distributed {
 namespace autograd {
 
-constexpr int kContextIdBits = 48;
-constexpr int64_t kContextIdMask = (1LL << kContextIdBits) - 1;
+constexpr int kAutoIncrementBits = 48;
+constexpr int64_t kAutoIncrementMask = (1LL << kAutoIncrementBits) - 1;
```

Comment:
So, the reason for this rename, is that the mask is used both for context ids, and autograd message ids?